app/forms/proof_form.rb

```diff
@@ -20,7 +20,7 @@ def resource
 
   def _save
     proof_type = DIDWW::Resource::ProofType.load(id: self.proof_type_id)
-    entity = DIDWW::Resource::Identity.load(id: self.identity_id)
+    entity = self.entity.constantize.load(id: self.entity_id)
```

Comment:
I think we should validate `entity` (which should be renamed to `entity_type`) and also hide class names from html

entity_type should be `Identity` or `Address` and we should validate that only such values are allowed and than transform them into classses